src/System.Management.Automation/CoreCLR/CorePsAssemblyLoadContext.cs

```diff
@@ -1,19 +1,14 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-windows`
Can load a native assembly.Can load native dll
```powershell
Exception calling "LoadNative" with "0" argument(s): "An attempt was made to load a program with an incorrect format. (0x8007000B)"
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Basic\Assembly.LoadNative.Tests.ps1: line 116
116:         [TestNativeClass2]::LoadNative() | Should -Not -BeNullOrEmpty

```